projects/languages/ros/webots_ros/src/complete_test.cpp

```diff
@@ -2067,6 +2067,48 @@ int main(int argc, char **argv) {
 
   get_max_torque_client.shutdown();
   time_step_client.call(time_step_srv);
+
+  ros::ServiceClient set_motor_feedback_client;
+  webots_ros::set_int motor_feedback_srv;
+  ros::Subscriber sub_motor_feedback_32;
+  set_motor_feedback_client =
+    n.serviceClient<webots_ros::set_int>(model_name + "/rotational_motor/torque_feedback_sensor/enable");
+
+  ros::ServiceClient sampling_period_motor_feedback_client;
+  webots_ros::get_int sampling_period_motor_feedback_srv;
+  sampling_period_motor_feedback_client =
+    n.serviceClient<webots_ros::get_int>(model_name + "/rotational_motor/torque_feedback_sensor/get_sampling_period");
+
+  motor_feedback_srv.request.value = 32;
```

Comment:
Where does this 32 value come from? Does it refer to a 32-bit float?

Reply:
No, it is just the sensors sampling period.